content/webapp/components/ContentPage/ContentPage.tsx

```diff
@@ -71,11 +71,25 @@ const ShameWhatWeDoHack = () => (
       secondaryLabels={[]}
       description={pageDescriptions.userPanel}
       Image={
-        <Image
-          contentUrl={`https://images.prismic.io/wellcomecollection/65334f9d-50d0-433f-a4ac-a780eef352e3_user_research_square.jpg?auto=compress,format`}
-          width={3200}
-          height={3200}
-          alt={''}
+        <PrismicImage
+          image={{
+            // We intentionally omit the alt text on promos, so screen reader
+            // users don't have to listen to the alt text before hearing the
+            // title of the item in the list.
+            //
+            // See https://github.com/wellcomecollection/wellcomecollection.org/issues/6007
+            contentUrl:
+              'https://images.prismic.io/wellcomecollection/65334f9d-50d0-433f-a4ac-a780eef352e3_user_research_square.jpg',
```

Comment:
Do we have a good place for hard-coded image URLs to live? It looks like `backgrounds.ts` is maybe it, or maybe something alongside `hardcoded-ids.ts`?

Doesn't need fixing in this PR, but since you're looking at image stuff is one to consider.